plane/source/docs/soaring.rst

```diff
@@ -130,14 +132,15 @@ Set loiter radius and bank angle limit
 
 The parameter :ref:`WP_LOITER_RAD<WP_LOITER_RAD>` sets how tight the loiter circle is. For thermalling it is usually
 best to have the aircraft fly at a 30 - 45 degree bank angle. The corresponding loiter radius can be calculated as 
-about airspeed squared over 10 (for 45 degrees) or 6 (for 30 degrees), from the equation
+about airspeed squared over ~10 (for 45 degrees) or ~6 (for 30 degrees), from the equation
 
 .. raw:: html
 
    <a href="https://www.codecogs.com/eqnedit.php?latex=r&space;=&space;\frac{v^2}{g&space;\tan&space;\phi}" target="_blank"><img src="https://latex.codecogs.com/gif.latex?r&space;=&space;\frac{v^2}{g&space;\tan&space;\phi}" title="r = \frac{v^2}{g \tan \phi}" /></a>
 
-You should make sure that the limiting bank angle :ref:`LIM_ROLL_CD<LIM_ROLL_CD>` is set a bit larger to give some room
-for manoeuvring.
+If using degrees for the tangent, the result will be in degrees, or radians if that is used. g  = 9.81 m/s/s, velocity (v) is in m/s. For example, if the airspeed in loiter is 20m/s, then the :ref:`WP_LOITER_RAD<WP_LOITER_RAD>` should be 40 for a 45 degree bank, assuming that :ref:`LIM_ROLL_CD<LIM_ROLL_CD>` is set at 45000 or higher.
```

Comment:
- It's not correct to say that the result will be in degrees; the result is in length units.
- Extra zero in LIM_ROLL_CD example, should be 4500 for 45 degrees

Reply:
will be fixed...

Reply:
fixed